tests/switchmap_/dashboard/table/test_event.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+from unittest.mock import MagicMock
+from flask_table import Table
+from switchmap.dashboard.table.events import _RawCol, EventTable, EventsRow, table
+
+
+class TestEventTable(unittest.TestCase):
+    def test_raw_col_td_format(self):
+        """Test the _RawCol td_format method."""
+        raw_col = _RawCol("")
+        self.assertEqual(raw_col.td_format("<b>test</b>"), "<b>test</b>")
+
+    def test_event_row_initialization(self):
+        """Test initialization of EventsRow."""
+        row_data = ["data0", "data1", "data2", "data3", "data4", "data5"]
+        row = EventsRow(row_data)
+        self.assertEqual(row.col0, "data0")
+        self.assertEqual(row.col5, "data5")
+
+    def test_event_table_creation(self):
+        """Test creation of EventTable with rows."""
+        row_data = [["data0", "data1", "data2", "data3", "data4", "data5"]]
+        rows = [EventsRow(data) for data in row_data]
+        table = EventTable(rows)
+        self.assertIsInstance(table, Table)
+        self.assertEqual(len(table.items), 1)
+
+    def test_table_function_no_events(self):
+        """Test table function with no events."""
+        events = []
+        result = table(events)
+        self.assertFalse(result)
+
+    def test_table_function_with_events(self):
```

Comment:
1. Test the table(events) with a list of EventMeta objects versus a mock. That is what is sent to the function.
2. Ensure you get a list of EventTable rows with specific values